bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -150,6 +150,54 @@ specify spatially variable soil thickness. If not present, use bottom
 of soil column (nlevsoi).
 </entry>
 
+<entry id="snicar_numrad_snw" type="integer" category="clm_physics"
+       group="snicar_inparm" valid_values="5,480" value="5" >
+number of wavelength bands used in SNICAR snow albedo calculation
+<default>Default: 5</default>
```

Comment:
Let's remove the `<default>` lines here unless they aren't set in the namelist_defaults and only in the FORTRAN code. I think all of these are set in the namelist defaults file so they should all be removed.